pkg/testutils/mock_config.go

```diff
@@ -14,11 +15,34 @@ const (
 	namespace     = "kubevirt"
 )
 
-func NewFakeClusterConfig(cfgMap *v1.ConfigMap) (*virtconfig.ClusterConfig, cache.SharedIndexInformer) {
+func NewFakeClusterConfig(cfgMap *v1.ConfigMap) (*virtconfig.ClusterConfig, cache.SharedIndexInformer, cache.SharedIndexInformer) {
 	informer, _ := NewFakeInformerFor(&v1.ConfigMap{})
```

Comment:
now when we have two informers, could you rename this one to cmInformer?

Reply:
yep, good idea. i'll make this change

Reply:
i won't push for it, just want to note that it wasn't changed @davidvossel

Reply:
oh! actually i did change something related to this, but in the config-map.go file instead of the mock one.

I'll make this change as well. I didn't look closely enough at where the comment was actually being made

Reply:
alrighty, should actually be fixed now :) thanks for the review